auth/src/main/java/software/amazon/awssdk/auth/signer/internal/AbstractAws4Signer.java

```diff
@@ -55,7 +57,8 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(Aws4Signer.class);
     private static final int SIGNER_CACHE_MAX_SIZE = 300;
-    private static final FifoCache<SignerKey> SIGNER_CACHE = new FifoCache<>(SIGNER_CACHE_MAX_SIZE);
+    private static final FifoCache<software.amazon.awssdk.auth.signer.SignerKey> SIGNER_CACHE =
```

Comment:
What's up with the FQCN? There may be a duplicate class somewhere.

Reply:
Yep, removed the duplicate.